doc/classes/CollisionPolygon2D.xml

```diff
@@ -19,6 +19,10 @@
 		<member name="one_way_collision" type="bool" setter="set_one_way_collision" getter="is_one_way_collision_enabled" default="false">
 			If [code]true[/code], only edges that face up, relative to [CollisionPolygon2D]'s rotation, will collide with other objects.
 			[b]Note:[/b] This property has no effect if this [CollisionPolygon2D] is a child of an [Area2D] node.
+			[b]Note:[/b] The one way collision direction can be configured by setting [member one_way_collision_direction].
+		</member>
+		<member name="one_way_collision_direction" type="Vector2" setter="set_one_way_collision_direction" getter="get_one_way_collision_direction" default="Vector2(0, 1)">
+			The direction used for one-way collision.
```

Comment:
I understand why it is a normalized vector like this, but I would expect this property to be exposed as a rotation in radians (displayed as degrees in the Inspector), which an appropriate name change. It's not very friendly, otherwise.

Reply:
I wrote about this in the proposal (https://github.com/godotengine/godot-proposals/issues/12093). The intention was to align with CharacterBody2D's up_direction field.